crates/runner-host/src/main.rs

```diff
@@ -35,9 +36,28 @@ fn with_state<R>(f: impl FnOnce(&mut board::State) -> R) -> R {
     f(STATE.lock().unwrap().as_mut().unwrap())
 }
 
+#[derive(Parser)]
+struct Flags {
+    #[cfg(feature = "web")]
+    #[clap(flatten)]
+    web_options: WebOptions,
+}
+
+#[derive(clap::Args)]
+struct WebOptions {
+    /// Host to start the webserver.
+    #[clap(long, default_value = "127.0.0.1")]
+    web_host: String,
+    /// Port to start the webserver.
+    #[clap(long, default_value = "5000")]
+    web_port: usize,    
+}
+
+
 #[tokio::main]
 async fn main() -> Result<()> {
     env_logger::init();
+    let _flags = Flags::parse();
```

Comment:
There are a few options to deal with conditional warnings:
- As you did, by ignoring it in all cases by changing the variable name.
- Ignoring it in all cases with a `#[allow(unused_variables)]` (or whatever the lint is).
- Ignoring it in cases where it's legitimately unused with `#[cfg_attr(not(feature = "web"), allow(unused_variables)]`.

Even if it's verbose, I would still suggest the latter option because it describes intent and is precise.